api/zsock.api

```diff
@@ -126,6 +126,16 @@
         <argument name = "endpoint" type = "string" />
     </constructor>
 
+    <constructor name = "new gather" state = "draft">
+        Create a GATHER socket. Default action is bind.
+        <argument name = "endpoint" type = "string" />
+    </constructor>
+
+    <constructor name = "new scatter" state = "draft">
+        Create a GATHER socket. Default action is connect.
```

Comment:
@sappo Oops I just saw this! I went and pulled @somdoron's latest, should have let you do it. :(